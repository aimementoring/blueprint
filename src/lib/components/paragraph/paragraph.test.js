import React from 'react';
import renderer from 'react-test-renderer';
import Paragraph from './title.js.js';

describe('Paragraph', () => {
  it('renders properly', () => {
    const tree = renderer.create(<Paragraph>Gradient default Paragraph</Paragraph>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('is backwards compatible (text passed as prop)', () => {
    const tree = renderer.create(<Paragraph text="Gradient default Paragraph" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
