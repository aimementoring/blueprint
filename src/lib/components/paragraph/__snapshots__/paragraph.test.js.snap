// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph is backwards compatible (text passed as prop) 1`] = `
<div
  className="theme-portal"
>
  <p
    className="paragraph "
  >
    Gradient default Paragraph
  </p>
</div>
`;

exports[`Paragraph renders properly 1`] = `
<div
  className="theme-portal"
>
  <p
    className="paragraph "
  >
    Gradient default Paragraph
  </p>
</div>
`;
